spec/concepts/todo/create_spec.rb

```diff
@@ -1,7 +1,30 @@
 require 'rails_helper'
 
 RSpec.describe Todo::Create do
-  it "can be spec'd" do
-    raise "Implement me"
+  
+  subject { described_class }
+
+  describe "Process" do 
+  	context "with valid params" do
+  		let(:params) { { todo: { title: "Todo title"  } } }
+
+  		it "persists the params" do
+  			res, op = subject.run(params)
+
+  			expect(op.model.persisted?).to be_truthy
+  			expect(res).to be_truthy
+  		end
+  	end
+
+  	context "with invalid params" do
+  		let(:params) { { todo: { title: "" } } }
+
+  		it "does not persist the params" do
+  			res, op = subject.run(params)
+
+  			expect(op.model.persisted?).to be_falsy
```

Comment:
`falsy` vs `false` and `truthy` vs `true`. I fell for this one once but one of our Syd devs told me we use `truthy/falsy` for `object present / nil` and `true/false` for boolean values.